src/base/orgs/Create.svelte

```diff
@@ -126,10 +126,10 @@
     <span slot="body">
       <div class="fields">
         <span class="label">Address</span>
-        <span><Address address={org.address} {config} /></span>
+        <span><Address noAvatar address={org.address} {config} /></span>
```

Comment:
I also wonder here why we are changing this?

Reply:
Since we are showing only the address there without reference to owners or users I added the `noAvatar` attribute.
But it's also the address of the orgs, which is kind of a edge case imo.